__tests__/initial.migration.test.ts

```diff
@@ -167,7 +167,7 @@ describe('database initial migration tests', () => {
           encoding: 'utf-8',
         })
         expect(signedMessage).toEqual(
-          'vzDocUViJh7ooOCZ-jBHKZddEsTa4yClHwhIL9SHJwjAv3bC6TZIcUnX36ZqNBWvLbnNAQvdtzqrVf3l0pv3QQ',
```

Comment:
JWT signing for ES256K now uses canonical signatures, with a low `S` value (through the use of `@noble/curves`)
This eliminates signature malleability, just like what happens for ethereum signatures, but it also means tests will change.